core/templates/dev/head/css/oppia.css

```diff
@@ -415,20 +415,25 @@ textarea {
 .oppia-navbar-breadcrumb {
   color: #fff;
   cursor: default;
-  font-size: 16px;
-  font-weight: 400;
-  line-height: 20px;
-  padding-top: 17px;
-  padding-bottom: 15px;
+  font-family: "Capriola", "Roboto", Arial, sans-serif;
+  font-size: 20px;
+  padding-top: 13px;
 }
 
 .oppia-navbar-breadcrumb-separator {
-  padding-right: 7px;
+  margin-right: 8px;
 }
 .oppia-navbar-breadcrumb-separator:after {
   content: ">";
 }
 
+.oppia-navbar-breadcrumb-icon {
+  cursor: pointer;
```

Comment:
Can the cursor: pointer go on the surrounding 'li', so that the cursor changes exactly at the same time when the tooltip appears?